src/transforms/combine-latest-with.ts

```diff
@@ -14,6 +14,17 @@
 import { Transform, Observable } from "../types.js";
 import { combineLatest } from "../combiners/combine-latest.js";
 
+/**
+ * Combines items from the original observable with the other observables.
+ * The resulting `Transform` emits items as array tuples.
+ * whenever the original or any of the given observables emit.
+ * The tuples contain the last emitted item from each observable.
```

Comment:
I was wondering if, for the sake of deduplication, we should remove the documentation for these transforms and just use `@link` to reference the top-level combiner? WDYT?

Reply:
Sounds good! I simplified the comments here, but I kept the parameters since they're slightly different compared to `combineLatest`/`zipWith`.